package.json

```diff
@@ -8,6 +8,8 @@
     "watch": "ts-node-dev --poll --no-notify src/server.ts",
     "test": "echo \"Error: no test specified\" && exit 1",
     "tsc": "tsc",
+    "knex": "knex migrate:latest",
+    "migrate": "knex migrate:latest",
```

Comment:
two commands that do the exact same thing?

there is nothing on the readme about this commands